UPGRADING.md

```diff
@@ -1,6 +1,51 @@
 Upgrading
 =========
 
+## 7.x to 8.x
+
+### node
+
+This version contains major improvements to the node notifier, including making the top-level `Bugsnag` static interface context-aware and enabling breadcrumbs for node projects.
+
+The minimum supported node version is now v12.17.0.
+
+#### context-aware `Bugsnag` calls
+
+When using `plugin-express`, `plugin-koa`, `plugin-restify`, or `plugin-contextualize` a clone of the top-level Bugsnag client is made so that any subsequent changes made to the client (such as attaching metadata) only affect the scope of a particular context (a request in the case of the web server plugins, and a function call in the case of `plugin-contextualize`).
```

Comment:
My feeling is that the "request context" is the overwhelming majority use case.  So might be better to rephrase as
```suggestion
When using `plugin-express`, `plugin-koa`, `plugin-restify`, or `plugin-contextualize` a clone of the top-level Bugsnag client is made so that any subsequent changes made to the client (such as attaching metadata) only affect the scope of the current web request. (Or a function call when using `plugin-contextualize`.)
```